submissions/TatyanaMolchanova/js-dom/script.js

```diff
@@ -0,0 +1,103 @@
+    },
+    'tigers': {
+        'menuTitle': 'Tigers',
+        'class': 'tigers',
+        'title': 'Tigers are awesome',
+        'info': 'There are two recognized subspecies of tiger*: the continental (Panthera tigris tigris) and the Sunda (Panthera tigris sondaica). The largest of all the Asian big cats, tigers rely primarily on sight and sound rather than smell for hunting. They typically hunt alone and stalk prey. A tiger can consume more than 80 pounds of meat at one time. On average, tigers give birth to two to four cubs every two years. If all the cubs in one litter die, a second litter may be produced within five months.',
+        'image': 'tigers'
+    },
+    'hares': {
+        'menuTitle': 'Hares',
+        'class': 'hares',
+        'title': 'Hares are awesome',
+        'info': 'Hares and jackrabbits are leporids belonging to the genus Lepus. Hares are classified in the same family as rabbits. They are similar in size and form to rabbits and have similar herbivorous diets, but generally have longer ears and live solitarily or in pairs. Also unlike rabbits, their young are able to fend for themselves shortly after birth rather than emerging blind and helpless. Most are fast runners. Hare species are native to Africa, Eurasia and North America.',
+        'image': 'hare'
+    },
+    'elephants': {
+        'menuTitle': 'Elephants',
+        'class': 'elephants',
+        'title': 'Elephants are awesome',
+        'info': 'Elephants are mammals of the family Elephantidae and the largest existing land animals. Three species are currently recognised: the African bush elephant, the African forest elephant, and the Asian elephant. Elephantidae is the only surviving family of the order Proboscidea; extinct members include the mastodons. The family Elephantidae also contains several now-extinct groups, including the mammoths and straight-tusked elephants. African elephants have larger ears and concave backs, whereas Asian elephants have smaller ears, and convex or level backs. Distinctive features of all elephants include a long proboscis called a trunk, tusks, large ear flaps, massive legs, and tough but sensitive skin. The trunk is used for breathing, bringing food and water to the mouth, and grasping objects. Tusks, which are derived from the incisor teeth, serve both as weapons and as tools for moving objects and digging. The large ear flaps assist in maintaining a constant body temperature as well as in communication. The pillar-like legs carry their great weight.',
+        'image': 'elephant'
+    },
+}
+
+const menu = document.createElement('ul'),
+    title = document.querySelector('.main h2'),
+    info = document.querySelector('.main p'),
+    image = document.querySelectorAll('.main img');
+
+function createMenu() {
+    const aside = document.querySelector('aside');
+
+    menu.classList.add('menu');
+
+
+    menu.innerHTML = `<li class="menu__item active" data-item="${content.cats.class}">${content.cats.menuTitle}</li>
```

Comment:
Next time you can use loop in cases like this. It's no need to do it by hand. Just iterate through your list with the data and generate some content using `forEach`, `map`, `reduce` etc.

Reply:
Ok, I used for in for creation li.